ckanext/xloader/plugin.py

```diff
@@ -21,6 +20,12 @@
     def config_declarations(cls):
         return cls
 
+if toolkit.check_ckan_version(min_version='2.11'):
```

Comment:
2.11 or 2.10? 
is 2.11 current master ckan?

Reply:
@duttonw yeah it is `2.11`. The IDatadictionaryForm only exists in master so far